.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Summary
+<!-- Provide a short description of the changes you have made here -->
+
+## Issue ticket number and link
+- Issue #[xyz]
```

Comment:
This could just mention and link to GitHub's linking system

They can just type a number after #